internal/common/conversion/type_conversion.go

```diff
@@ -62,3 +62,8 @@ func IsStringPresent(strPtr *string) bool {
 func MongoDBRegionToAWSRegion(region string) string {
 	return strings.ReplaceAll(strings.ToLower(region), "_", "-")
 }
+
+// AWSRegionToMongoDBRegion converts region in us-east-1-like format to US_EAST_1-like
+func AWSRegionToMongoDBRegion(region string) string {
```

Comment:
`conversion.ValRegion` does something similar, is it possible to unifiy both functions so we only have one logic to transform between region formats?